src/Inciter/Partitioner.h

```diff
@@ -1260,6 +1265,7 @@ class Partitioner : public CBase_Partitioner< HostProxy,
                         CkCallback(CkReductionTarget(Transporter,load), host));
     }
     //! Compute average communication cost of merging the linear system
+    //! \param[in] host Host to signal to
```

Comment:
I'm not sure if I understand your question. The reason why this is done this way, is explained starting at line [1209](https://github.com/quinoacomputing/quinoa/blob/mms/src/Inciter/Partitioner.h#L1209).